game_stats_scrapper.py

```diff
@@ -29,7 +29,7 @@ def get_match_count(soup: bs4.BeautifulSoup) -> int:
     return int(match_count)
 
 
-def get_general_data(data: bs4.element.Tag) -> str:
+def get_general_data(data: bs4.element.Tag, active_url: str) -> str:
```

Comment:
This change should be in last commit, in the future I shouldn't adding change like this at different bigger commit.